testdata/ci/runner/Jenkinsfile

```diff
@@ -197,7 +194,7 @@ spec:
                             dir('testdata/ci/components') {
                                 sh """#!/usr/bin/env bash
                                     source /usr/local/lib/yorc-ci-utils.sh
-                                    forgeCompsURL="\$(getURLFromPart "https://ystia.jfrog.io/ystia/binaries/ystia/forge/dist/${FORGE_COMPS_BRANCH}" 'all-types-[0-9].*?.zip')"
+                                    forgeCompsURL="\$(getURLFromPart "https://ystia.jfrog.io/ystia/binaries/ystia/forge/dist/develop" 'all-types-3.*?.zip')"
```

Comment:
I think the `3` is not necessary as `getURLFromPart` takes the latest one and it may be confusing in the future

Reply:
ah ok good, I will remove this change